### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by anmolbabu**
Signed-off-by: anmolbabu <anmolbudugutta@gmail.com>

### CODE REVIEW

**Comment by anmolbabu** [comment]
@shtripat @r0h4n @nthomas-redhat Please review

File: tendrl/performance_monitoring/central_store/__init__.py [hunk 1]
```diff
@@ -122,14 +122,12 @@ def get_node_summary(self, node_ids=None):
...
-                del current_node_summary['enabled']
-                del current_node_summary['obj_value']
-                del current_node_summary['flows']
-                del current_node_summary['atoms']
+                if '_etcd_cls' in current_node_summary:
+                    del current_node_summary['_etcd_cls']
+                if 'value' in current_node_summary:
+                    del current_node_summary['value']
+                if '_defs' in current_node_summary:
+                    del current_node_summary['_defs']
```
Comment (r0h4n) [inline tendrl/performance_monitoring/central_store/__init__.py]: add del current_node_summary['list']

Comment (anmolbabu) [inline tendrl/performance_monitoring/central_store/__init__.py]: No that's not appearing as in 
[{"cpu_usage": {"percent_used": "35.79", "updated_at": "2017-03-23T18:05:15.869834"}, "memory_usage": {"total": "3973312512.0", "used": "276865024.0", "percent_used": "6.97", "updated_at": "2017-03-23T18:05:15.951245"}, "storage_usage": {"total": "37295452160.0", "percent_used": "5.16843147451", "used": "1927589888.0", "updated_at": "2017-03-23T18:05:16.140431"}, "alert_count": 0, "node_id": "584e7a89-1216-4c09-9e83-265231921166"}, {"cpu_usage": {"percent_used": "34.17", "updated_at": "2017-03-23T18:01:02.770698"}, "memory_usage": {"total": "8201162752.0", "used": "223502336.0", "percent_used": "3.73", "updated_at": "2017-03-23T18:02:02.924727"}, "storage_usage": {"total": "55832629248.0", "percent_used": "3.45387918301", "used": "1928400896.0", "updated_at": "2017-03-23T18:03:02.906718"}, "alert_count": 0, "node_id": "3fb70b17-d1e9-4f52-97a9-ff445999fbf5"}, {"cpu_usage": {"percent_used": "66.03", "updated_at": "2017-03-23T18:05:15.880104"}, "memory_usage": {"total": "1927004160.0", "used": "209514496.0", "percent_used": "10.87", "updated_at": "2017-03-23T18:05:15.979560"}, "storage_usage": {"total": "28701138944.0", "percent_used": "6.71216423766", "used": "1926467584.0", "updated_at": "2017-03-23T18:05:16.188720"}, "alert_count": 0, "node_id": "b63b4487-6455-4561-bd08-f96217d03c97"}]

Comment (anmolbabu) [inline tendrl/performance_monitoring/central_store/__init__.py]: The actual object.__dict__: 

{'_etcd_cls': <class 'tendrl.performance_monitoring.objects.summary._PerformanceMonitoringSummary'>, 'cpu_usage': {'percent_used': u'35.79', 'updated_at': u'2017-03-23T18:05:15.869834'}, 'memory_usage': {'total': u'3973312512.0', 'used': u'276865024.0', 'percent_used': u'6.97', 'updated_at': u'2017-03-23T18:05:15.951245'}, 'storage_usage': {'total': u'37295452160.0', 'percent_used': u'5.16843147451', 'used': u'1927589888.0', 'updated_at': u'2017-03-23T18:05:16.140431'}, 'alert_count': 0, 'value': 'monitoring/summary/nodes/584e7a89-1216-4c09-9e83-265231921166', '_defs': NamedDict({'attrs': {'storage_usage': {'type': 'dict'}, 'cpu_usage': {'type': 'dict'}, 'memory_usage': {'type': 'dict'}, 'node_id': {'type': 'str'}, 'alert_count': {'type': 'int'}}, 'help': 'Node Summary', 'obj_list': 'monitoring/summary/nodes', 'enabled': True, 'obj_value': 'monitoring/summary/nodes/$PerformanceMonitoringSummary.node_id', 'flows': {}, 'atoms': {}}), 'node_id': '584e7a89-1216-4c09-9e83-265231921166'}

**Comment by anmolbabu** [comment]
@r0h4n @shtripat @nthomas-redhat Please review

File: tendrl/performance_monitoring/central_store/__init__.py (hunk 1 above)
Comment (r0h4n) [inline tendrl/performance_monitoring/central_store/__init__.py]: Yes, I agree its not there now, but it "value" and "list" are something that we use to GET and LIST objects in etcd

**Pull request merged**